motoman_gp25_support/package.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp25_support</name>
+  <version>0.3.5</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman gp25 (and variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman gp25 manipulators.
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>gp25 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information 
+      found in the online 
+      https://www.motoman.com/hubfs/Robots/gp25.pdf
```

Comment:
The link to this document did not work for me

Reply:
Yes, our website was recently redone.  I just replace the robot model of an older model with GP25 without paying attention to it.  I'll correct the link to:
https://www.motoman.com/en-us/products/robots/industrial/assembly-handling/gp-series/gp25

Reply:
@EricMarcil: I would actually recommend we start noting document numbers/IDs and revisions here, instead of linking to sites.

If (or most likely *when*) the Motoman site gets updated again, all links in package manifests run the risk of going stale again and I doubt we'll pay enough attention to catch that.

I've done this in the HC10 support package: